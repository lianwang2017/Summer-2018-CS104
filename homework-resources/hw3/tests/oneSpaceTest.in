CS Minorville, BUAD
Mike Zyda, CSGM
Joe Genius, PHCS