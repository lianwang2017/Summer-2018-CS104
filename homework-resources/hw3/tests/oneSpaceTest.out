BUAD 
CS Minorville

CSGM 
Mike Zyda

PHCS 
Joe Genius

